dockstore-client/src/test/java/io/dockstore/client/cli/YamlValidatorTest.java

```diff
@@ -248,6 +231,46 @@ void dotGitHubDirectory() {
             assertEquals(errorMsg, ex.getMessage());
         }
     }
+
+    @Test
+    void invalidSubclass() {
+        String testDirectory = "src/test/resources/YamlVerifyTestDirectory/invalid-subclass";
+        try {
+            YamlVerifyUtility.dockstoreValidate(testDirectory);
```

Comment:
Should `assert.fail()` after this line otherwise the test will still pass if `dockstoreValidate` doesn't throw an exception

Reply:
Good catch, thanks!

I have now fixed it. [a41e0b2](https://github.com/dockstore/dockstore-cli/pull/235/commits/a41e0b2d1ab0dbdff009c88e867b84e9052ff712)